### ISSUE TITLE
Evaluation of workflow engines

### ISSUE THREAD

**Issue by rabernat** [ocean-transport/science-together#1]
Here is a big list of different workflow engines: https://github.com/meirwah/awesome-workflow-engines. (I just proposed to [add Parsl](https://github.com/meirwah/awesome-workflow-engines/pull/86).)

We should make some attempt to evaluate the options systematically.

**Comment by jbusecke** [comment]
Good idea. I will try to first define our [wishlist](https://github.com/ocean-transport/science-together#feature-wishlist) a bit more clearly and then maybe have a table with ✅ and ❌?
